shell/platform/darwin/ios/framework/Source/FlutterEmbedderKeyResponder.mm

```diff
@@ -632,9 +623,9 @@ - (void)sendPrimaryFlutterEvent:(const FlutterKeyEvent&)event
   _responseId += 1;
   uint64_t responseId = _responseId;
   FlutterKeyPendingResponse* pending =
-      [[[FlutterKeyPendingResponse alloc] initWithHandler:self responseId:responseId] autorelease];
+      [[FlutterKeyPendingResponse alloc] initWithHandler:self responseId:responseId];
   [callback pendTo:_pendingResponses withId:responseId];
-  _sendEvent(event, HandleResponse, pending);
+  _sendEvent(event, HandleResponse, (void* _Nullable)CFBridgingRetain(pending));
```

Comment:
We had some tests that crashed with a `__bridge` cast here, the compiler suggested a `CFBridgingRetain` instead.